core/src/main/java/bisq/core/network/p2p/inventory/messages/GetInventoryResponse.java

```diff
@@ -28,11 +28,11 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import lombok.EqualsAndHashCode;
-import lombok.Value;
+import lombok.Getter;
+import lombok.ToString;
 
-@Value
-@EqualsAndHashCode(callSuper = false)
```

Comment:
EqualsAndHashCode should be left

Reply:
commit 0c9c96165b2df15d9bd4c8a671d3083192f11403